desktop/cmp/form/FormField.ts

```diff
@@ -253,17 +264,20 @@ const editableChild = hoistCmp.factory<FieldModel>({
         disabled,
         displayNotValid,
         leftErrorIcon,
-        commitOnChange
+        commitOnChange,
+        testId
     }) {
         const {props} = child;
+        testId = props.testId ?? testId;
```

Comment:
Should this be done on line 280 (where it is assigned to `overrides`) so it is symmetrical with the `disabled` override?